{
  "Header": {
    "Original": "CVE-2016-8614 (A flaw was found in Ansible before version 2.2.0. The apt_key module d ...)",
    "Line": 436647,
    "ID": "CVE-2016-8614",
    "Description": "(A flaw was found in Ansible before version 2.2.0. The apt_key module d ...)"
  },
  "Annotations": [
    {
      "Original": "- ansible 2.2.0.0-1 (bug #842984)",
      "Line": 436648,
      "Type": "package",
      "Package": "ansible",
      "Kind": "fixed",
      "Version": "2.2.0.0-1",
      "BugNo": 842984
    },
    {
      "Original": "[jessie] - ansible \u003cnot-affected\u003e (Vulnerable code introduced later)",
      "Line": 436649,
      "Type": "package",
      "Release": "jessie",
      "Package": "ansible",
      "Kind": "not-affected",
      "Description": "Vulnerable code introduced later"
    },
    {
      "Original": "NOTE: Fixed upstream in v2.2.0.0-1",
      "Line": 436650,
      "Type": "NOTE",
      "Description": "Fixed upstream in v2.2.0.0-1"
    },
    {
      "Original": "NOTE: https://github.com/ansible/ansible-modules-core/issues/5237",
      "Line": 436651,
      "Type": "NOTE",
      "Description": "https://github.com/ansible/ansible-modules-core/issues/5237"
    },
    {
      "Original": "NOTE: https://github.com/ansible/ansible-modules-core/pull/5353",
      "Line": 436652,
      "Type": "NOTE",
      "Description": "https://github.com/ansible/ansible-modules-core/pull/5353"
    },
    {
      "Original": "NOTE: https://github.com/ansible/ansible-modules-core/pull/5357",
      "Line": 436653,
      "Type": "NOTE",
      "Description": "https://github.com/ansible/ansible-modules-core/pull/5357"
    }
  ]
}